### ISSUE TITLE
broker account creation - rejected  employment_status (with a correct enum)

### ISSUE THREAD

**Issue by radzzii** [alpacahq/alpaca-py#212]
if we include the employment_status (with a correct enum), it will be rejected with message: "value is not a valid enumeration member"

No issue by API

<img width="1399" alt="Screen Shot 2022-11-16 at 10 40 05" src="https://user-images.githubusercontent.com/5517785/202315375-74e708a4-0e87-40b4-8ce4-78b3da2e07bc.png">

the code in Google Colab: 
https://colab.research.google.com/drive/1XO0lEAqKsro_1i6et3B0lgUZmud61vxl

the code to reproduce:
```
!pip install -q alpaca-py==0.6.0 
import requests
import alpaca.broker as brokerapi
from numpy import trace
import pandas as pd
from alpaca.broker.requests import CreateAccountRequest
from alpaca.broker.client import BrokerClient
from alpaca.broker.models import (
                        Contact,
                        Identity,
                        Disclosures,
                        Agreement
                    )
from alpaca.broker.enums import TaxIdType, FundingSource, AgreementType

api_id='' 
api_key=''

client = brokerapi.BrokerClient(api_id, api_key)

# Contact
contact_data = Contact(
            email_address="test41421aa4141@example.com",
            phone_number="555-666-7788",
            street_address=["20 N San Mateo Dr"],
            city="San Mateo",
            state="CA",
            postal_code="94401",
            country="USA"
            )
# Identity
identity_data = Identity(
        given_name="John",
        middle_name="Smith",
        family_name="Doe",
        date_of_birth="1990-01-01",
        tax_id="666-55-4321",
        tax_id_type=TaxIdType.USA_SSN,
        country_of_citizenship="USA",
        country_of_birth="USA",
        country_of_tax_residence="USA",
        funding_source=[FundingSource.EMPLOYMENT_INCOME]
        )



# Agreements
agreement_data = [
    Agreement(
      agreement=AgreementType.MARGIN,
      signed_at="2020-09-11T18:09:33Z",
      ip_address="185.13.21.99",
    ),
    Agreement(
      agreement=AgreementType.ACCOUNT,
      signed_at="2020-09-11T18:13:44Z",
      ip_address="185.13.21.99",
    ),
    Agreement(
      agreement=AgreementType.CUSTOMER,
      signed_at="2020-09-11T18:13:44Z",
      ip_address="185.13.21.99",
    ),
    Agreement(
      agreement=AgreementType.CRYPTO,
      signed_at="2020-09-11T18:13:44Z",
      ip_address="185.13.21.99",
    )
]

# Disclosures
disclosure_data = Disclosures(
        is_control_person=False,
        is_affiliated_exchange_or_finra=False,
        is_politically_exposed=False,
        immediate_family_exposed=False,
        employment_status='employed'
        )

account_data = CreateAccountRequest(
                        contact=contact_data,
                        identity=identity_data,
                        disclosures=disclosure_data,
                        agreements=agreement_data
                        )
account = client.create_account(account_data)
print(account)
```

**Comment by sshcli** [comment]
Try with `alpaca-py==0.6.1` to see if it works with the latest version

**Comment by radzzii** [comment]
thanks @sshcli 
I just tried it with 0.6.1, same issue

**Comment by radzzii** [comment]
but when i tried to re-send the payload, i got email validated message: `an account with the requested email address already exists`

**Comment by radzzii** [comment]
i checked the broker dashboard, the account successfully created, but i don't get the account_id return from the create_account()

**Comment by sshcli** [comment]
> "value is not a valid enumeration member"

A similar issue have been solved by @OlegRa in PR https://github.com/alpacahq/alpaca-py/pull/182
Basically he just added a new OrderStatus `HELD = "held"` and the issue has been solved.


This was the error btw:
```
1 validation error for Order
legs -> 0 -> status
  value is not a valid enumeration member; permitted: 'new', 'partially_filled', 'filled', 'done_for_day', 'canceled', 'expired', 'replaced', 'pending_cancel', 'pending_replace', 'accepted', 'pending_new', 'accepted_for_bidding', 'stopped', 'rejected', 'suspended', 'calculated' (type=type_error.enum; enum_values=[<OrderStatus.NEW: 'new'>, <OrderStatus.PARTIALLY_FILLED: 'partially_filled'>, <OrderStatus.FILLED: 'filled'>, <OrderStatus.DONE_FOR_DAY: 'done_for_day'>, <OrderStatus.CANCELED: 'canceled'>, <OrderStatus.EXPIRED: 'expired'>, <OrderStatus.REPLACED: 'replaced'>, <OrderStatus.PENDING_CANCEL: 'pending_cancel'>, <OrderStatus.PENDING_REPLACE: 'pending_replace'>, <OrderStatus.ACCEPTED: 'accepted'>, <OrderStatus.PENDING_NEW: 'pending_new'>, <OrderStatus.ACCEPTED_FOR_BIDDING: 'accepted_for_bidding'>, <OrderStatus.STOPPED: 'stopped'>, <OrderStatus.REJECTED: 'rejected'>, <OrderStatus.SUSPENDED: 'suspended'>, <OrderStatus.CALCULATED: 'calculated'>])
```

I hope the above information helps with the troubleshooting process

**Comment by br-follow** [comment]
I am still seeing this issue

```
'Failed to create alpaca account for account.id=UUID(\'49f38966-04d9-4ab3-97bc-297c11a4e203\') due to e=ValidationError(model=\'ParsingModel[Disclosures]\', errors=[{\'loc\': (\'__root__\', \'employment_status\'), \'msg\': "value is not a valid enumeration member; permitted: \'unemployed\', \'employed\', \'student\', \'retired\'", \'type\': \'type_error.enum\', \'ctx\': {\'enum_values\': [<EmploymentStatus.UNEMPLOYED: \'unemployed\'>, <EmploymentStatus.EMPLOYED: \'employed\'>, <EmploymentStatus.STUDENT: \'student\'>, <EmploymentStatus.RETIRED: \'retired\'>]}}])'
```

**Comment by br-follow** [comment]
@haxdds - could you please take a look? We are unable to open accounts

**Comment by haxdds** [comment]
Closed by #235
